Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -130,10 +130,10 @@ extension Reducer {
       func assert(step: Step) {
         switch step.type {
         case let .send(action, update):
-          self.send(action, file: step.file, line: step.line, update)
+          self.send(action, update, file: step.file, line: step.line)
```

Comment:
Trailing closures used to not work when default arguments came after them, but that was fixed a long time ago, so I think it should be mostly safe to reorder them. I doubt many folks depend on `file:` and `line:` externally, and those that do should get a handy fix-it whenever they upgrade.